src/libs/ValidationUtils.js

```diff
@@ -75,32 +75,38 @@ function isValidSSNLastFour(ssnLast4) {
 function isValidIdentity(identity) {
     if (!isValidAddress(identity.street)) {
         showBankAccountFormValidationError(translateLocal('bankAccount.error.address'), true);
+        showErrorModal();
         return false;
     }
 
     if (identity.state === '') {
         showBankAccountFormValidationError(translateLocal('bankAccount.error.addressState'), true);
+        showErrorModal();
         return false;
     }
 
     if (!isValidZipCode(identity.zipCode)) {
         showBankAccountFormValidationError(translateLocal('bankAccount.error.zipCode'), true);
+        showErrorModal();
         return false;
     }
 
     if (!isValidDate(identity.dob)) {
         showBankAccountFormValidationError(translateLocal('bankAccount.error.dob'), true);
+        showErrorModal();
         return false;
     }
 
     if (!isValidSSNLastFour(identity.ssnLast4)) {
         showBankAccountFormValidationError(translateLocal('bankAccount.error.ssnLast4'), true);
```

Comment:
We are still passing the `shouldGrowl` argument in all of these.

Reply:
ops, good catch